xlators/cluster/dht/src/dht-rebalance.c

```diff
@@ -2874,7 +2874,10 @@ gf_defrag_migrate_single_file(void *opaque)
 
             LOCK(&defrag->lock);
             {
-                defrag->total_failures += 1;
+                if (fop_errno == EBUSY)
```

Comment:
@chen1195585098, thanks for the fix.
Two comments:
1. If rebalance is running due to remove-brick and this file located on a removed brick, it should be counted as a failure.
2. On this case the above logging (L2871) can be changed to `migrate-data skipped...`.

Reply:
I agree tamar comments.

Reply:
@tshacked @mohit84 ,thanks for your comments,
I will double-check the process based on these suggestions.